src/quo/components/wallet/summary_info/view.cljs

```diff
@@ -51,6 +52,18 @@
          :amount  (str (:amount arbitrum) " " (or (:token-symbol arbitrum) "ARB"))
          :theme   theme}])]))
 
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:theme [:schema.common/theme]]
+      [:type [:keyword]]
+      [:account-props [:map]]
```

Comment:
If you're not narrowing down the schema, it's enough to just use it without the extra [] around it e.g. `[:type :keyword]`. Otherwise the schema would be invalid.